application/view/common/site-list-entry.phtml

```diff
@@ -1,7 +1,11 @@
 <?php
+$thumbnail = $showThumbnail ? $site->thumbnail() : null;
 $summary = $showSummary ? $site->summary() : null;
 ?>
 <div class="site">
+    <?php if ($thumbnail): ?>
+    <p class="site-thumbnail"><?php echo $this->thumbnail($thumbnail, 'square', ['class' => 'site-thumbnail-image']); ?></p>
```

Comment:
A nitpick, but a `div` wrapper would be more appropriate as a wrapper for an image.